api/control/v1alpha1/types.go

```diff
@@ -260,11 +260,19 @@ type BindingSpec struct {
 type DownsyncObjectReferences struct {
 	// `clusterScope` holds a list of cluster-scoped object references with their associated
 	// resource versions to downsync.
-	ClusterScope []ClusterScopeDownsyncObject `json:"clusterScope,omitempty"`
+	ClusterScope []ClusterScopeDownsyncObjectAndStatusCollectors `json:"clusterScope,omitempty"`
 
 	// `namespaceScope` holds a list of namespace-scoped object references with their associated
 	// resource versions to downsync.
-	NamespaceScope []NamespaceScopeDownsyncObject `json:"namespaceScope,omitempty"`
+	NamespaceScope []NamespaceScopeDownsyncObjectAndStatusCollectors `json:"namespaceScope,omitempty"`
+}
+
+// NamespaceScopeDownsyncObjectAndStatusCollectors represents a specific namespace-scoped object to downsync,
```

Comment:
s/represents/references/

Reply:
I see that the comment on ClusterScopeDownsyncObjectAndStatusCollectors got changed, but the comment here still says "represents".

Reply:
@MikeSpreitzer done